ortc.html

```diff
@@ -799,6 +803,10 @@
                     An <code><a>RTCIceTransport</a></code> instance is constructed (optionally) from an <code><a>RTCIceGatherer</a></code>.
                     If <var>gatherer.state</var> is "closed" or <var>gatherer.component</var> is "RTCP", then throw an <code>InvalidStateError</code> exception.
                 </p>
+                <p>
+                    An <code><a>RTCIceTransport</a></code> object in the "closed" state can be garbage-collected
+                    when it is no longer referenced.
```

Comment:
Currently, the non-normative ICE state diagram in Section 3.6 shows a potential transition from "failed" to "checking" or "disconnected".